ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.18.yaml

```diff
@@ -90,13 +90,15 @@ tests:
     workflow: openshift-ci-security
 - always_run: false
   as: aws-e2e-operator
+  skip_if_only_changed: ^(?:docs|\.github|renovate\.json|\.tekton)/|\.md$|^(?:\.gitignore|renovate\.json|OWNERS|PROJECT|LICENSE)$
```

Comment:
should be  the same as the others: `^(?:docs|\.github|\.tekton)/|\.md$|^(?:\.gitignore|renovate\.json|OWNERS|PROJECT|LICENSE)$`

If there is a another file with renovate on both sides of the regex thats my fault :sweat_smile:

Reply:
nope, the rest are good. The culprit; I was reading an outdated file.

Reply:
Fixed in https://github.com/openshift/release/commit/7e11fe613250797a499008c9376e7cd251ab35a1, PTAL